identity_iota_core/src/rebased/migration/identity.rs

```diff
@@ -343,17 +349,69 @@ pub async fn get_identity(
     return Ok(None);
   };
 
-  let did = IotaDID::from_alias_id(&object_id.to_string(), client.network());
-  let (id, multi_controller, created, updated) = match unpack_identity_data(&did, &data)? {
-    Some(data) => data,
-    None => {
-      return Ok(None);
-    }
+  let content = data
```

Comment:
Was this due to a merge conflict or is there a reason, we can't keep

```rs
  let did = IotaDID::from_alias_id(&object_id.to_string(), client.network());
  let Some((id, multi_controller, created, updated)) = unpack_identity_data(&did, &data)? else {
    return Ok(None);
  };

  let did_doc =
    IotaDocument::from_iota_document_data(multi_controller.controlled_value(), true, &did, created, updated)
      .map_err(|e| Error::DidDocParsingFailed(e.to_string()))?;
```

Also, it seems another merge conflict undid the `let Some(...) = ... else {};` change we did for a previous commit, so the version in the code block above is the intended version. >.>

Reply:
Yeah merge issues :(

Reply:
Fixed